achievements/member.achievement.js

```diff
@@ -0,0 +1,31 @@
+var moment = require('moment');
+
+var member = {
+  name: 'Member?',
+  check: function(pullRequest, shall) {
+    if (isWaitingLongTime(pullRequest)) {
+
+      var achieve = {
+        avatar : 'images/achievements/member.achievement.jpg',
+        name : 'Member?',
```

Comment:
don't you want to add the pull request in the name?

like: `name: 'Member pull request #' + pullRequest.number`?